crates/proto/src/quic/quic_stream.rs

```diff
@@ -161,7 +161,7 @@ impl QuicStream {
         let bytes = self.receive_bytes().await?;
         let message = Message::from_vec(&bytes)?;
 
-        // assert that the message id is 0, this is a bad dns-over-quic packet if not
+        // assert that the message id is 0, this is a bad quic packet if not
```

Comment:
This should probably be left as-is, since the requirement comes from RFC 9250, not RFC 9000.
```suggestion
        // assert that the message id is 0, this is a bad dns-over-quic packet if not
```